urllib3/util/retry.py

```diff
@@ -273,10 +273,18 @@ def _is_read_error(self, err):
         """
         return isinstance(err, (ReadTimeoutError, ProtocolError))
 
+    def _is_method_retryable(self, method):
+        """ Is this method retryable? (Based on method whitelist)
+        """
+        if self.method_whitelist and method.upper() not in self.method_whitelist:
+            return False
+
+        return True
+
     def is_retry(self, method, status_code, has_retry_after=False):
         """ Is this method/status code retryable? (Based on method/codes whitelists)
```

Comment:
We're going to need to clear up the fact that these two methods have very similar docstrings.